spec/active_operation/pipeline_spec.rb

```diff
@@ -111,6 +125,26 @@ def execute
     it { is_expected.to utilize_operations(string_generator, string_capitalizer) }
   end
 
+  context(
+    "when composed of two operations, one that generates a string and one that lowercases strings by receiving its " \
+    "input as a required property"
+  ) do
+    subject(:composed_operation) do
+      operations = [string_generator, string_lowercaser_by_property]
+
+      Class.new(described_class) do
+        use operations.first
+        use operations.last, text: -> { previous_output }
```

Comment:
What I wanted is to support passing on pipeline arguments to an underlying operation:

```ruby
class Pipeline < ActiveOperation::Pipeline
  property! :transformation_method, accepts: [:lowercase, :uppercase], default: :lowercase

  use StringGenerator
  use StringTransformator, method: -> { transformation_method }
end
```